specification/translation/data-plane/Azure.AI.DocumentTranslation/preview/2023-11-01-preview/openapi.json

```diff
@@ -0,0 +1,481 @@
+          },
+          {
+            "name": "X-ClientTraceId",
+            "in": "header",
+            "description": "A client-generated GUID to uniquely identify the request.",
+            "required": false,
+            "type": "string",
+            "x-ms-client-name": "clientTraceId"
+          },
+          {
+            "name": "If-None-Match",
+            "in": "header",
+            "description": "Etag comparison, if it is provided and its value matches the response then request will return Status => 304 (Not modified).",
+            "required": false,
+            "type": "string",
+            "x-ms-client-name": "ifNoneMatch"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Response for the Document Formats.",
+            "schema": {
+              "type": "array",
+              "items": {
+                "$ref": "#/definitions/FileFormat"
+              },
+              "x-ms-identifiers": []
+            },
+            "headers": {
+              "X-RequestId": {
+                "type": "string",
+                "description": "Value generated by the service to identify the request. It is used for troubleshooting purposes."
+              },
+              "ETag": {
+                "type": "string",
+                "description": "This will have value of response hash, customer can provide this value for any consecutive request as “If-None-Match” header. To make sure get response if the value is changed otherwise 304."
+              }
+            }
+          },
+          "default": {
+            "description": "An unexpected error response.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            },
+            "headers": {
+              "X-RequestId": {
+                "type": "string",
+                "description": "Value generated by the service to identify the request. It is used for troubleshooting purposes."
+              },
+              "ETag": {
+                "type": "string",
+                "description": "This will have value of response hash, customer can provide this value for any consecutive request as “If-None-Match” header. To make sure get response if the value is changed otherwise 304."
+              }
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Document formats": {
+            "$ref": "./examples/DocumentFormats_MaximumSet_Gen.json"
+          }
+        }
+      }
+    },
+    "/document:translate": {
+      "post": {
+        "operationId": "SingleTranslateDocument",
+        "description": "Use this API to translate a single document.\nAll supported document formats can be obtained with formats API.\nSingle Glossary / Translation memory can be included in the request and applied by the service when the document is translated.",
+        "produces": [
+          "application/octet-stream",
+          "application/json"
+        ],
```

Comment:
Under what conditions will this operation return application/octet-stream?

Reply:
This likely from this `DocumentTranslateResult | ErrorResponse`, which means
- success would be octet-stream
- error would be JSON